firefox_ui_tests/functional/locationbar/test_access_locationbar.py

```diff
@@ -27,6 +26,7 @@ def setUp(self):
         self.autocomplete_results = self.locationbar.autocomplete_results
         self.urlbar = self.locationbar.urlbar
 
+    @skip_under_xvfb
     def test_access_locationbar_history(self):
```

Comment:
This test is failing right now. We should better mark it as skipped in the manifest file for now and figure out a proper solution for it. Can you add this, so I could land it later? Otherwise I could also add this little addition.